src/quicktype-core/language/Python.ts

```diff
@@ -232,6 +232,24 @@ function snakeNameStyle(version: PythonVersion, original: string, uppercase: boo
     );
 }
 
+function threeToTwo(input: Sourcelike): Sourcelike {
+    // @ts-ignore
+    const [loose, types] = (input as any[]).reduce(([p, f], e) => {
```

Comment:
You can either do `[loose, _types]` or `[loose, ]` and you won't need the `@ts-ignore`.

Reply:
First alternative gave same error, second alternative worked. Done.